src/sentry/static/sentry/app/views/settings/components/forms/datePickerField.tsx

```diff
@@ -0,0 +1,111 @@
+import 'react-date-range/dist/styles.css';
+import 'react-date-range/dist/theme/default.css';
+
+import {Calendar} from 'react-date-range';
+import React from 'react';
+import moment from 'moment';
+import styled from '@emotion/styled';
+
+import {inputStyles} from 'app/styles/input';
+import DropdownMenu from 'app/components/dropdownMenu';
+import {IconCalendar} from 'app/icons';
+import space from 'app/styles/space';
+
+import InputField, {onEvent} from './inputField';
+
+type Props = Omit<InputField['props'], 'field'>;
+
+function handleChangeDate(
+  onChange: onEvent,
+  onBlur: onEvent,
+  date: Date,
+  close: Function
```

Comment:
```suggestion
  close: () => void
```

Reply:
I was inferring it from `actions.close` which provides it as `Function`

Reply:
oh I see... yeah, in this case, better use Function